change/@fluentui-api-docs-7a2a50f4-a2f4-4356-b4df-8a56d0b2dab5.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Upgrade to typescript 4.1.5",
+  "packageName": "@fluentui/api-docs",
+  "email": "joschect@microsoft.com",
```

Comment:
nit: perhaps might want to update the email field ?

Reply:
I've made changefiles with the correct address for work I've done, but I think this is still the right attribution for this changefile.